File: UniversalVariables.vb
Imports MySql.Data.MySqlClient

Module UniversalVariables
    Public conn As New MySqlConnection
    Public ds As New DataSet
    Public cmd As New MySqlCommand
    Public dr As MySqlDataReader
    Public sql As String

    Public conn1 As New MySqlConnection
    Public ds1 As New DataSet
    Public cmd1 As New MySqlCommand
    Public dr1 As MySqlDataReader
    Public sql1 As String

    Public conn2 As New MySqlConnection
    Public ds2 As New DataSet
    Public cmd2 As New MySqlCommand
    Public dr2 As MySqlDataReader
    Public sql2 As String

    Public Sub connect()
        cmd.CommandText = sql
        cmd.CommandType = CommandType.Text
        cmd.Connection = conn
        conn.Open()
        dr = cmd.ExecuteReader
    End Sub

    Public Sub connectAlt()
        cmd1.CommandText = sql1
        cmd1.CommandType = CommandType.Text
        cmd1.Connection = conn1
        conn1.Open()
        dr1 = cmd1.ExecuteReader
    End Sub

    Public Sub connectAgain()
        cmd2.CommandText = sql2
        cmd2.CommandType = CommandType.Text
        cmd2.Connection = conn2
        conn2.Open()
        dr2 = cmd2.ExecuteReader
    End Sub

End Module

File: UniversalVariables.cs
using System.Data;
using MySql.Data.MySqlClient;

namespace MilkTeaOrderAndInventorySystem
{

    static class UniversalVariables
    {
        public static MySqlConnection conn = new MySqlConnection();
        public static DataSet ds = new DataSet();
        public static MySqlCommand cmd = new MySqlCommand();
        public static MySqlDataReader dr;
        public static string sql;

        public static MySqlConnection conn1 = new MySqlConnection();
        public static DataSet ds1 = new DataSet();
        public static MySqlCommand cmd1 = new MySqlCommand();
        public static MySqlDataReader dr1;
        public static string sql1;

        public static MySqlConnection conn2 = new MySqlConnection();
        public static DataSet ds2 = new DataSet();
        public static MySqlCommand cmd2 = new MySqlCommand();
        public static MySqlDataReader dr2;
        public static string sql2;

        public static void connect()
        {
            cmd.CommandText = sql;
            cmd.CommandType = CommandType.Text;
            cmd.Connection = conn;
            conn.Open();
            dr = cmd.ExecuteReader();
        }

        public static void connectAlt()
        {
            cmd1.CommandText = sql1;
            cmd1.CommandType = CommandType.Text;
            cmd1.Connection = conn1;
            conn1.Open();
            dr1 = cmd1.ExecuteReader();
        }

        public static void connectAgain()
        {
            cmd2.CommandText = sql2;
            cmd2.CommandType = CommandType.Text;
            cmd2.Connection = conn2;
            conn2.Open();
            dr2 = cmd2.ExecuteReader();
        }

    }
}